### DISCUSSION TITLE
Access Denied Error while Writing to eMMC

### DISCUSSION THREAD

**Discussion by erentkll** [polarfire-soc/polarfire-soc-documentation#324] [HSS and U-Boot] [answered]
Hi, I have an application and I want to write it to eMMC in order to run the application on boot. I generated the payload and tried to write it to eMMC using win32DiskImager. However, when I select partition :/G and try to write the payload, I get "Error 5: Access is denied" and the text "An error ocurred when attenpting to get a handle on the volume". How can I solve this? Thank you in advance.
Regards,

**Comment by hughbreslin** [comment, marked as answer]
Hey @erentkll 

have you tried deleting all of the existing eMMC partitions (if you haven't programmed the eMMC before it came pre-programmed with Linux) and formatting the drive before using win32 disk imager? 

You can delete the partitions using computer management -> storage -> disk management  on windows, right click on each partition on your eMMC (be very careful not to pick the wrong drive!!) and delete it. Once you've done this select the unallocated space and select "New simple volume" and format as fat32. Try writing you .bin again after that :)

![image](https://github.com/polarfire-soc/polarfire-soc-documentation/assets/44862881/0554ca18-8925-4905-9c6c-dddbb35b0522)

You will need admin privilege's for all of this (including disk imager) btw. Some people who have had issues writing with Win32 disk imager have had success with different tools like Etcher for example
